release_build_versions.txt

```diff
@@ -4,6 +4,7 @@
 # The below lists _additional_ kubernetes versions to be built.
 
 kubernetes-v1.28.5 # required for CAPO CI
+kubernetes-v1.27.3 # required for CAPG CI
```

Comment:
I suppose the pinning is required on the fixed version (1.27.16 was the last iteration).